sktime/utils/utilities.py

```diff
@@ -0,0 +1,160 @@
+import numpy as np
+from pandas import DataFrame
```

Comment:
With `pandas`, I would always use `import pandas as pd` and then `pd.DataFrame`, that seems to be the convention